src/interfaces.ts

```diff
@@ -0,0 +1,6 @@
+export interface FirebaseAppConfig {
```

Comment:
Add comment todo to remove this interface once the main typings for Firebase include a better definition than `Object`.